src/mmw/js/src/user/views.js

```diff
@@ -219,6 +239,14 @@ var LoginModalView = ModalBaseView.extend({
 
     // Login with ITSI
     itsiLogin: function() {
+        var selector = '.itsi-login',
+            model = this.model;
+
+        if (coreUtils.modalButtonDisabled(model, selector)) {
+            return;
+        }
+        coreUtils.modalButtonToggle(model, selector, false);
```

Comment:
This logic is slightly different than all of the other additions (it lacks an else statement). Is that intentional?